test/docker_container_test_case.py

```diff
@@ -81,22 +85,32 @@ def _create_container(cls, image: str, container_port: int, **kwargs) -> Netloc:
                                                ports=ports,
                                                **kwargs)
         try:
+            time_it = datetime.now()
```

Comment:
```suggestion
            start = datetime.now()
```
This should be moved above the call to `cls._docker.containers.run()` so that even if sleep() is not called in the while loop, a time greater than `0.00` will be reported.